### ISSUE TITLE
Converting caffe model to Keras

### ISSUE THREAD

**Issue by hiiroo** [HyeonwooNoh/DeconvNet#12]
I was trying to convert deconvnet caffe model to keras with tensorflow backend. I only transpose convolutional layer weights since I thought caffe and tensorflow use correlation. However I believe I've got stuck with probably batch normalization layers. I'm extracting bn weights but I don't know if they are required to be rotated or something else when using with keras batch normalization layer? Until now I've tried to use them by reshaping and setting to gamma and beta variables at keras batch norm layer.
